sympy/series/tests/test_order.py

```diff
@@ -405,3 +405,7 @@ def test_order_subs_limits():
 
     assert Order(x**2).subs(x, y - 1) == Order((y - 1)**2, (y, 1))
     assert Order(10*x**2, (x, 2)).subs(x, y - 1) == Order(1, (y, 3))
+
+
+def test_issue_9192():
+    assert O(1)*O(1) == O(1)
```

Comment:
Should that be part of this PR? As thetest title is named after the issue, it should probably test all the facets of the issue (at least if they're related). In this case they don't appear to be, so perhaps change the test title?